### PR TITLE
Fix dev ops publish bug

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Move `Publish to DevOps Feed` step into loop to iteratively publish individual artifacts to fix https://dev.azure.com/azure-sdk/internal/_build/results?buildId=552610&view=logs&j=b1e79959-24d8-5aa9-2799-72d40c3e051b&t=4b34df18-a0a9-5141-3485-374c7e76236f

### CODE REVIEW

File: sdk/eventgrid/Azure.Messaging.EventGrid/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Release History
 ## 4.0.0-beta.3 (Unreleased)
```
Comment (weshaggard) [inline sdk/eventgrid/Azure.Messaging.EventGrid/CHANGELOG.md:3]: Looks unrelated.

**Review by weshaggard — APPROVED** [review]

File: sdk/eventgrid/Azure.Messaging.EventGrid/CHANGELOG.md (hunk 1 above)
Comment (chidozieononiwu) [reply sdk/eventgrid/Azure.Messaging.EventGrid/CHANGELOG.md:3]: I fixed that spacing as part of a rebase which is why it showed up.
